data/hashing.ts

```diff
@@ -7,49 +7,51 @@
  * @resource {https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest} MDN: Cryptographic Digests
  *
  * Hashing data is a common operation that is facilitated
- * through Deno's support of the Web Crypto API. There are
- * some more advanced uses that may require extensions to
- * the api that are provided by the standard library.
+ * through Deno's support for the Web Crypto API. In addition,
+ * Deno's implementation extends the standard API, allowing for
+ * more advanced uses.
  */
 
-// Our original plaintext message
+// The standard library has extensions to the Web
+// Crypto API that are useful when doing things
+// like hashing a file. These can be accessed through the
+// "crypto" module, a drop-in replacement for the Web Crypto
+// API that delegates to the native implementation when
+// possible.
```

Comment:
Though I moved the `import` to the top of the code, I left the bit about hashing a file. 

If needed, that sentence can be moved closer to actual file hashing operation.